apps/src/templates/manageStudents/ManageStudentsTable.story.jsx

```diff
@@ -6,9 +6,11 @@ import manageStudents from './manageStudentsRedux';
 import {Provider} from 'react-redux';
 
 const initialState = {
-  loginType: '',
-  studentData: {},
-  sectionId: null,
+  manageStudents: {
```

Comment:
This fix isn't related to HMR, it was just logging an error to the console any time you opened storybook:
![screenshot from 2018-02-01 13-55-31](https://user-images.githubusercontent.com/1070243/35705563-a8855ffc-0757-11e8-8c85-cd201d6d3bf7.png)